src/components/HomePage/index.js

```diff
@@ -12,103 +12,168 @@ import List from "@material-ui/core/List";
 import { userList } from "./userList";
 import useStyles from "./styles";
 import SideBar from "../SideBar/index";
+import TagCard from "../CardTabs/Tags/index";
+import EventsCard from "../CardTabs/Events/index";
+import OrgUser from "../../assets/images/org-user.svg";
+import UserCard from "../CardTabs/Users/index";
 
 function HomePage({ background = "white", textColor = "black" }) {
   const classes = useStyles();
-    const [value, setValue] = useState(2);
-    const [footerContent, setFooterContent] = useState([
-        { name: "Help", link: "https://dev.codelabz.io/" },
-        { name: "About", link: "https://dev.codelabz.io/" },
-        { name: "Content Policy", link: "https://dev.codelabz.io/" },
-        { name: "Terms", link: "https://dev.codelabz.io/" },
-        { name: "Privacy Policy", link: "https://dev.codelabz.io/" },
-        { name: "CodeLabz @2022", link: "https://dev.codelabz.io/" },
+  const [value, setValue] = useState(2);
+  const [footerContent, setFooterContent] = useState([
+    { name: "Help", link: "https://dev.codelabz.io/" },
+    { name: "About", link: "https://dev.codelabz.io/" },
+    { name: "Content Policy", link: "https://dev.codelabz.io/" },
+    { name: "Terms", link: "https://dev.codelabz.io/" },
+    { name: "Privacy Policy", link: "https://dev.codelabz.io/" },
+    { name: "CodeLabz @2022", link: "https://dev.codelabz.io/" },
```

Comment:
shall we change this to 
```
name: `CodeLabz @${new Date().getFullYear()}`
```

in this way the year will be changed accordingly.

Reply:
@Niweera sure, I will do it.